src/main/java/microsoft/exchange/webservices/data/core/request/ResolveNamesRequest.java

```diff
@@ -36,7 +36,7 @@
 import microsoft.exchange.webservices.data.enumeration.ServiceErrorHandling;
 import microsoft.exchange.webservices.data.enumeration.XmlNamespace;
 import microsoft.exchange.webservices.data.exception.ServiceXmlSerializationException;
-import microsoft.exchange.webservices.data.interfaces.ILazyMember;
+import microsoft.exchange.webservices.data.core.ILazyMember;
```

Comment:
hmm I did this because `IAsyncResult`has only 1 usage in implements clause and that one is located in the misc package and `ILazyMember` has no usage in implements clause but 5 usages in new anonymous inheritor creation which are located in core.
Since this was some kind of guessing which could be the best package are you suggesting to put them together to core/misc?